test/e2e/common/secrets.go

```diff
@@ -48,6 +48,27 @@ var _ = framework.KubeDescribe("Secrets", func() {
 		doSecretE2EWithMapping(f, &mode)
 	})
 
+	It("should be able to mount in a volume regardless of a different secret existing with same name in different namespace", func() {
+		var (
+			namespace2  *api.Namespace
+			err         error
```

Comment:
That would limit the scope.
I need namespace2 and err in the following lines.

https://groups.google.com/forum/#!topic/golang-nuts/v6Xm9C7ENgc